src/lib/components/events/event-item.svelte

```diff
@@ -119,12 +117,11 @@
     display: none;
   }
 
-  .event-body a {
-    text-decoration: none;
+  .event-body .event-name {
     transition: all 0.15s ease-in-out;
   }
 
-  .event-body a:hover {
+  .event-body .event-name:hover {
```

Comment:
I think we should do `.event-body:hover .event-name` so the event name changes color when the card is hovered on top. It just makes more sense, since the fake link appearance doesn't actually behave like a link anyway.

Reply:
I see how that can make more sense. I updated the CSS a little in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/280/commits/60fea4f7cf182160593af2cdf97c3cd72c39557b> to achieve this.